ElectronicObserverTypes/Attacks/Specials/ColoradoSpecialAttack.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using ElectronicObserverTypes.Extensions;
+
+namespace ElectronicObserverTypes.Attacks.Specials;
+
+public record ColoradoSpecialAttack : SpecialAttack
+{
+	public ColoradoSpecialAttack(IFleetData fleet) : base(fleet)
+	{
+	}
+
+	public override string GetDisplay() => AttackResources.SpecialColorado;
+
+	public override bool CanTrigger()
+	{
+		List<IShipData?> ships = Fleet.MembersInstance.ToList();
+
+		if (!ships.Any()) return false;
+
+		IShipData? flagship = ships.First();
+		if (flagship is null) return false;
+		if (flagship.MasterShip.ShipClassTyped is not ShipClass.Colorado) return false;
+
+		if (flagship.HPRate <= 0.5) return false;
+
+		if (Fleet.NumberOfSurfaceShipNotRetreatedNotSunk() < 6) return false;
+
+		IShipData? firstHelper = ships[1];
+		if (firstHelper is null) return false;
```

Comment:
The 3 ifs for helper checks are duplicated. It might be better to extract them into a function named MeetsHelperRequirements or something like that. Do that for other special attacks too.

Reply:
Hopefully didn't miss anything while fixing